### PR TITLE
Fixing comment section

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
### Functionality:
fixing comment section

### Solution:
fixed the comment section in task details page which was taking empty comments using validation service.

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
try entering empty comment in the task details page

### CODE REVIEW

File: src/app/body/task-details/task-details.component.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ import { Activity } from 'src/app/Interface/ActivityInterface';
 import { UserServiceService } from 'src/app/services/user-service/user-service.service';
 import { ApplicationSettingsService } from 'src/app/services/applicationSettings/application-settings.service';
 import { StartServiceService } from 'src/app/services/start/start-service.service';
+import { ValidationService } from 'src/app/services/validation/validation.service';
+import { Condition } from 'selenium-webdriver';
```
Comment (Pendyala-thanvi) [inline src/app/body/task-details/task-details.component.ts:19]: Remove this import statement

Comment (abhishek-mishra019) [reply src/app/body/task-details/task-details.component.ts]: done

**Review by Pendyala-thanvi — APPROVED** [review]
Approved

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**